pkg/sdk/logging/api/v1beta1/logging_types.go

```diff
@@ -561,6 +563,16 @@ func (l *Logging) QualifiedName(name string) string {
 	return fmt.Sprintf("%s-%s", l.Name, name)
 }
 
+// ClusterDomainAsSuffix formats the cluster domain as a suffix, e.g.:
+// .Spec.ClusterDomain == "", returns ""
+// .Spec.ClusterDomain == "cluster.local", returns ".cluster.local"
+func (l *Logging) ClusterDomainAsSuffix() string {
+	if l.Spec.ClusterDomain == nil || *l.Spec.ClusterDomain == "" {
```

Comment:
Explicitly setting `ClusterDomain` to `""` should result in no suffix—as you wrote, but leaving it unset should result in the "default" `.cluster.local` suffix so that we don't introduce a breaking change. (Some people might rely on this suffix being set by default.)